app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -165,8 +165,12 @@ def create
     end
 
     logger.debug("Creating meeting #{params[:meetingID]} on BigBlueButton server #{server.id}")
-    # Pass along all params except the built in rails ones
-    uri = encode_bbb_uri('create', server.url, server.secret, pass_through_params)
+
+    # Get list of params that should not be modified by create API call
+    excluded_params = Rails.configuration.x.send('create_exclude_params')
```

Comment:
You don't need to use the `send` method to make a static method call, just do `Rails.configuration.x.create_exclude_params`